tools/eubfr-cli/bin/cli.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+const program = require('commander');
+const pkg = require('../package.json');
+
+// Utilities
+const getCredentials = require('../lib/getProducerCredentials');
+
+// Commands
+const upload = require('../commands/upload');
+const show = require('../commands/show');
+const deleteFile = require('../commands/delete');
+
+program.version(pkg.version).usage('[command] [option]');
+
+program
+  .command('upload <file> [moreFiles...]')
+  .description('Uploads a new file to the data lake')
+  .option('-p, --producer [producer]', "Producer's name. Defaults to 'agri'.")
+  .action((file, moreFiles, options) => {
+    // The `file` is required.
+    const files = [file];
+    const producer = options.producer || 'agri';
+    const credentials = getCredentials(producer);
+
+    if (moreFiles.length) {
+      files.push(...moreFiles);
+    }
+
+    upload({ files, credentials });
+  });
+
+program
+  .command('show [file]')
+  .description('Displays files of a given producer.')
+  .option('-p, --producer [producer]', "Producer's name. Defaults to 'agri'.")
+  .action((file, options) => {
+    const producer = options.producer || 'agri';
+
+    show({ file, producer });
+  });
+
+program
+  .command('delete [files...]')
+  .description('Deletes files by computedKey. Pass --confirm to delete all.')
+  .option('-p, --producer [producer]', "Producer's name. Defaults to 'agri'.")
```

Comment:
If the `computed_key` is `inforegio/something`, and nothing is passed as `--producer` (-> it defaults to "agri"), the file is still deleted, right? It seems a bit counterintuitive